src/main/kotlin/com/bardsoftware/papeeria/backend/cosmas/CosmasInMemoryService.kt

```diff
@@ -83,4 +87,23 @@ class CosmasInMemoryService : CosmasGrpc.CosmasImplBase() {
             this.files[request.fileId] = fileVersions
         }
     }
+
+    override fun createPatch(request: CosmasProto.CreatePatchRequest,
+                                        responseObserver: StreamObserver<CosmasProto.CreatePatchResponse>) {
+        println("Get request for create new history of file # ${request.fileId}")
+        synchronized(this.patches) {
+            val historyOfFile = patches[request.fileId] ?: mutableListOf()

```

Comment:
you really want to use the terminology consistently. If you access patches, you most likely get a patch or patch list.